lib/util/init.js

```diff
@@ -239,7 +240,7 @@ const Init = {
         }
 
         // create files before installing dependencies to ensure .gitignore is properly set up
-        status = await InitConfig.createIdeConfigFiles();
+        status = await InitConfig.createIdeConfigFiles(versionBeforeUpgrade);
```

Comment:
if only referenced once, then move versionBeforeUpgrade -> properties.version

Reply:
nope. it gets overwritten in the process of upgrade before we can do our checks. that's why we have to store a backup here